angular-portfolio/src/app/name/name.component.html

```diff
@@ -0,0 +1,8 @@
+<mat-card class="name">
+    <mat-card-title class="intro">
+        <h1 id="header">Billy Porter</h1>
```

Comment:
Use a consistent style when choosing between "id" or "class". Typically, class is used when it will be reused and id when it is unique. In practice at Google, we just always use class and never use id though.